packages/remix-dev/compiler/assets.ts

```diff
@@ -73,7 +73,10 @@ export async function createAssetsManifest(
     if (!output.entryPoint) continue;
 
     let entryPointFile = path.resolve(
-      output.entryPoint.replace(/(^browser-route-module:|\?browser$)/g, "")
+      output.entryPoint.replace(
+        /(^browser-route-module:|^pnp:|\?browser$)/g,
```

Comment:
I didn't expect to need to handle the `pnp` namespace here. Can you explain why we need to consider it here?

Reply:
Short version: without adding this, the PnP build fails with "Missing output for entry point"

I'm assuming that the plugin takes over all module resolutions, no matter if they use an absolute or relative path or if you try to import installed dependencies. It resolves everything into the `pnp` namespace, including the `entry.client.tsx` file.